### PR TITLE
Feat/server

### PR DESCRIPTION
**Pull request by Assafkatz99**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,15 +8,20 @@
     "@babel/core": "^7.21.8",
     "@babel/plugin-syntax-jsx": "^7.21.4",
```
Comment (orrosmanhls) [inline package.json]: Remove caret ("^") and tilda ("~")symbols

File: package.json [hunk 2]
```diff
@@ -47,6 +52,7 @@
     "eslint-plugin-react": "7.32.2",
     "eslint-plugin-react-hooks": "4.6.0",
     "jest-environment-jsdom": "^29.4.1",
+    "jest-environment-node": "^29.4.1",
     "jsdom": "~20.0.3",
     "nx": "16.1.4",
     "nx-cloud": "16.0.5",
```
Comment (orrosmanhls) [inline package.json]: remove

**Review by orrosmanhls — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Assafkatz99) [reply package.json]: resolved

File: package.json (hunk 2 above)
Comment (Assafkatz99) [reply package.json]: resolved

**Review by orrosmanhls — APPROVED** [review]

**Pull request merged**